spec/models/project_step_spec.rb

```diff
@@ -36,4 +36,20 @@
     expect(step[:original_date]).not_to be_nil
   end
 
+  context 'groups' do
```

Comment:
I have had success using this gem to avoid those extra db hits. https://github.com/pcreux/rspec-set